pkg/executor/docker/errors.go

```diff
@@ -0,0 +1,17 @@
+package docker
+
+import "github.com/bacalhau-project/bacalhau/pkg/models"
+
+const DOCKER_EXECUTOR_COMPONENT = "Executor/Docker"
```

Comment:
some errors will have `Docker` and others `Executor/Docker`, even though they are both related to Docker executor. Better to just use `Docker` for both

Reply:
Done.